### DISCUSSION TITLE
POOL ID

### DISCUSSION THREAD

**Discussion by alphatrade1** [atiselsts/uniswap-v3-liquidity-math#10] [Q&A]
When running 

# get pool info
try:
    variables = {"pool_id": POOL_ID}
    response = client.execute(gql(pool_query), variable_values=variables)

    if len(response['pools']) == 0:
        print("pool not found")
        exit(-1)

returns pool not found for the pool ID in the script:

# default pool id is the 0.3% USDC/ETH pool
POOL_ID = "0x8ad599c3a0ff1de082011efddc58f1908eb6e6d8"

What should be the pool id?

**Comment by atiselsts** [comment]
Hmm, is this Ethereum mainnet? Pool addresses will be different on other networks.

Other than that, it depends on which pool you want to access...

**Reply by alphatrade1** [reply]
I'm trying to run the script : https://github.com/atiselsts/uniswap-v3-liquidity-math/blob/master/subgraph-liquidity-range-example.py and that is the pool id in that script.

But as said getting the error with pool id not found. Hence what is the current pool id for ether uniswap v3  that I can use to run that script

**Reply by atiselsts** [reply]
I works for me, so either you have changed it locally, or I'd need more information to diagnose the problem.
